mutest.gemspec

```diff
@@ -39,8 +39,4 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency('anima',         '~> 0.3.0')
   gem.add_runtime_dependency('concord',       '~> 0.1.5')
   gem.add_runtime_dependency('regexp_parser', '~> 0.4.1')
-
-  gem.add_development_dependency('rubocop-rspec', '~> 1.10.0')
```

Comment:
Why is this being dropped here?

Also: Github won't show me a real diff for the Gemfile because it thinks it's a binary file... which is weird.